docs/userguide/packaging/index.md

```diff
@@ -127,14 +130,14 @@ updated.
 
 | File | Links to | Purpose |
 |------------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
-| libicuuc.so | libicuuc.so.54.7 | Required for link: Newly linked applications will follow this link, which should not cause any functional difference at link time. |
+| `libicuuc.so` | libicuuc.so.54.7 | Required for link: Newly linked applications will follow this link, which should not cause any functional difference at link time. |
 | libicuuc.so.54 | libicuuc.so.54.7 | Required for runtime: Because it now links to version .7, existing applications linked to version 5.4.3 will follow this link and use the 5.4.7 code. |
```

Comment:
Kindof a nit, but why does the top row get the backticks, but other rows don't?
(In the table above all the rows got the backtick) :)

Reply:
because auto-linkify (which is in some viewers but not Github itself) thinks it's a Somali website.

Reply:
:-)